### PR TITLE
WF-1242 Add sample implementation of PlayerIdentityToken verification code in authentication flow example

### PR DESCRIPTION
**Pull request by jasminequah**

### CODE REVIEW

**Comment by Helcaraxan** [comment]
Please also wait for review by someone from the Worker team for things touching the Platform SDK. I've asked @RNabel given I'm unavailable next week.

**Comment by jasminequah** [comment]
Thanks for the feedback Lan! I'm waiting to merge [this PR](https://github.com/improbable/everything/pull/4040) with the relevant updates to the documentation once this one goes through :)

File: examples/BYOAuthFlow/Program.cs [hunk 1]
```diff
@@ -103,6 +103,17 @@ protected override void Run()
                     ProjectName = ProjectName
                 });
 
+            Console.WriteLine("Verifying PlayerIdentityToken");
```
Comment (tbimprob) [inline examples/BYOAuthFlow/Program.cs]: This step needs to be added to the summary comment on lines 17-24

Comment (jasminequah) [reply examples/BYOAuthFlow/Program.cs]: I didn't notice that, now updated. Thanks!

File: examples/README.md [hunk 2]
```diff
@@ -7,6 +7,8 @@
 
 You can build and run the C# solution in an IDE, or using the .NET Core CLI.
 
+Building and uploading the assembly is not required. However, the assembly is contained within the repository, should you wish to run the deployment in your own project.
```
Comment (Helcaraxan) [inline examples/README.md]: Slightly rephrase this to be more explicit. Suggestion:
```markdown
Building and uploading the assembly of this example project is not required
to use the authentication logic shown here in your own project. However if
you wish to see this example in action for yourself you can use the pre-built
assembly included in this repository:

1. <steps to follow to upload the pre-built assembly and run a deployment>
2. ...
```

Comment (jasminequah) [reply examples/README.md]: No problem! Thanks for the feedback, I have added the changes.

File: examples/README.md [hunk 3]
```diff
@@ -18,6 +18,15 @@ dotnet restore
 dotnet run
 ```
 
+## Running an example in your own project
+It is not required to build and upload an assembly to run the solution, as they run in the `platform_sdk_examples` project which already contains the assembly.
```
Comment (Helcaraxan) [inline examples/README.md]: This is actually inducing people into mistakes as they will attempt to run the assembly in that given project which they do not have access to. It's better to simply explain that developers do not need to build & upload the example code to their own project in order to use the authentication code in their own game.

Comment (jasminequah) [reply examples/README.md]: My mistake, I did not realise people do not have access to the given project. Thank you for letting me know, I have clarified this part.

**Comment by Helcaraxan** [comment]
You'll need to rebase on top of master in order to merge though.

**Comment by jasminequah** [comment]
Closing this PR as changes have been merged [here](https://github.com/spatialos/platform-sdk-csharp/pull/41).
